roles/st2mistral/tasks/main.yml

```diff
@@ -85,18 +85,18 @@
 
 - name: Setup mistral DB tables, etc
   become: yes
-  shell: /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head && touch /etc/mistral/mistral-db-manage.upgrade.head.ansible.has.run
+  shell: /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head && touch /etc/mistral/mistral-db-manage.upgrade.head.{{ st2mistral_version }}.ansible.has.run
   args:
-    creates: /etc/mistral/mistral-db-manage.upgrade.head.ansible.has.run
+    creates: /etc/mistral/mistral-db-manage.upgrade.head.{{ st2mistral_version }}.ansible.has.run
```

Comment:
With the `st2mistral_version: latest` it will still run only once.

So different mechanism is needed.

Reply:
True. Hmm...

Reply:
I think the initial approach @bigmstone suggested, - to delete the lock file once the `install st2mistral` task has been changed.